rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -991,7 +991,13 @@ error_code sceNpTrophyUnlockTrophy(u32 context, u32 handle, s32 trophyId, vm::pt
 		return SCE_NP_TROPHY_ERROR_ALREADY_UNLOCKED;
 	}
 
-	ctxt->tropusr->UnlockTrophy(trophyId, 0, 0); // TODO: add timestamps
+	vm::var<CellRtcTick> tick;
+	if (error_code error = cellRtcGetCurrentTick(tick))
+	{
+		sceNpTrophy.error("sceNpTrophyUnlockTrophy: Failed to get timestamp: 0x%x", +error);
+	}
+
+	ctxt->tropusr->UnlockTrophy(trophyId, 0, tick->tick); // TODO: add timestamps
```

Comment:
This comment is not valid anymore, is it?

Reply:
Since I have no idea what the first part is, yes

Reply:
done